lib/http/2/header/huffman_statemachine.rb

```diff
@@ -268,7 +268,7 @@ class Huffman
         [[28, 29], [28, 5], [29, 29], [29, 5], [30, 29], [30, 5], [31, 29], [31, 5], [127, 29], [127, 5], [220, 29], [220, 5], [249, 29], [249, 5], [nil, 254], [nil, 255]],
         [[10, 17], [10, 18], [10, 19], [10, 20], [10, 21], [10, 22], [10, 23], [10, 7], [13, 17], [13, 18], [13, 19], [13, 20], [13, 21], [13, 22], [13, 23], [13, 7]],
         [[22, 17], [22, 18], [22, 19], [22, 20], [22, 21], [22, 22], [22, 23], [22, 7], [256, 17], [256, 18], [256, 19], [256, 20], [256, 21], [256, 22], [256, 23], [256, 7]],
-      ].each { |arr| arr.each { |subarr| subarr.each(&:freeze) }.freeze }.freeze
+      ].each { |arr| arr.each { |subarr| subarr.freeze }.freeze }.freeze
```

Comment:
Nice. This looks like it was a bug right? Only the arrays need to be frozen not values?

Reply:
exactly. not a "bug" as in, it didn't lead to breaking code, but it made this constant unusable across ractors.